README.md

```diff
@@ -0,0 +1,115 @@
+# What is Lyric Curator?
+A bot that looks for random lyrics of songs on the internet and post a part of it on social media. Made with python, this bot scrapes lyrics and automatically tweet it on the twitter page shown below. 
+
+[![official_website](https://www.linkpicture.com/q/Screenshot-2022-10-04-030204.png)](https://twitter.com/lyricurator)
+
+# Want to contribute?
+
+## Hacktoberfest 2022
+
+You can contribute to this project under [Hacktoberfest 2022](https://hacktoberfest.com/) 💫
+
+![image](https://user-images.githubusercontent.com/70385488/192114009-0830321a-d227-4a4d-8411-6c03b54d7ce6.png)
+
+Hacktoberfest 2022 is finally here, and we're pretty excited about it. We think it's a great time to encourage, recognise and reward developers in the community who dedicate their time to creating for others.
+
+Hacktoberfest is an annual festival hosted by DigitalOcean that takes place in the month of October to celebrate the open source community. As part of this 31-day celebration, contributors and open source maintainers are encouraged to work on open source projects and win swag to mark the occasion.
+
```

Comment:
Add a section here with a list of all the available CLI arguments and write 1 or 2 sentences for each one that explains what it does.
All the CLI arguments can be found in the file [console_args.py](https://github.com/MohamedYasser97/lyricurator/blob/master/console_args.py)
Add them and you're good to go

Reply:
okay got it